docs/src/components/pricing/PricingTable.tsx

```diff
@@ -284,7 +282,18 @@ function ColumnHead({
     </Typography>
   );
   return (
-    <Box sx={{ pl: nested ? 2.5 : 1, pr: 1, alignSelf: 'center', justifySelf: 'flex-start' }}>
+    <Box
+      sx={{
+        pl: 1,
+        pr: 1,
```

Comment:
```suggestion
        px: 1,
```

We can simplify this a bit!